lib/intrinsics/src/SPIRVIntrinsics.jl

```diff
@@ -32,6 +32,7 @@ macro import_all()
         # bring all the names of this module in scope
         name in (:SPIRVIntrinsics, :eval, :include) && continue
         startswith(string(name), "#") && continue
+        name in (:method_table, :@device_function, :@device_override) && continue
```

Comment:
😱 
That repo is hosted here; just remove the functions from there as well?

Reply:
So this is a good question. Inside SPIRVIntrinsics.jl we will likely want to use `device_function` and `device_override`. At least `device_function` makes sense. 

https://github.com/JuliaGPU/OpenCL.jl/blob/f7d5ae434acb13db4701366f1dfc09114e4e38e0/lib/intrinsics/src/atomic.jl#L19

For me that means we shouldn't remove these macros, we could rename them to `spirv_function` or somemthing similar.

Reply:
I think this also makes this a breaking change for SPIRVIntrinsics...